frb_codegen/src/generator/dart/ty_struct.rs

```diff
@@ -120,18 +125,26 @@ impl TypeDartGeneratorTrait for TypeStructRefGenerator<'_> {
                 .collect::<Vec<_>>();
             if has_methods {
                 constructor_params.insert(0, extra_argument);
+                if self.ir.is_exception {
+                    constructor_params.push("Backtrace? backtrace".to_string());
+                }
```

Comment:
Do we still need it? Since there is already a field named "backtrace" manually written by users already